includes/functions.php

```diff
@@ -31,6 +32,12 @@ function register_batch_process( $args ) {
 			$batch_process = new Users();
 			$batch_process->register( $args );
 			break;
+		case 'site':
+			if ( is_multisite() ) {
```

Comment:
For now, if you are not on multisite, the process never even gets registered. So it won’t show up in the admin at all and can’t be run.